src/Http/XmlResponse.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimpleSAML\Module\casserver\Http;
+
+use Symfony\Component\HttpFoundation\Response;
+
+class XmlResponse extends Response
+{
+    public function __construct(?string $content = '', int $status = 200, array $headers = [])
+    {
+        parent::__construct($content, $status, array_merge($headers, [
+            'Content-Type' => 'text/xml; charset=ISO-8859-1',
```

Comment:
The standards don't say anything about the charset, do they? I think it's better to use the default utf-8, which is already set in the xml-header.